decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -113,6 +113,25 @@ def reported_content_url
       def official?
         author.nil?
       end
+
+      # Public: The maximum amount of votes allowed for this proposal. 0 means infinite.
+      #
+      # Returns an Integer with the maximum amount of votes.
```

Comment:
`Returns an Integer with the maximum amount of votes if the feature is enabled, nil otherwise.` maybe?